controllers/constants/constants.go

```diff
@@ -19,4 +19,13 @@ const (
 	VCLUSTER_K8S_DISTRO         = "k8s"
 	K3S_DATASTORE_ENDPOINT      = "K3S_DATASTORE_ENDPOINT"
 	VCLUSTER_INGRESS_HOSTNAME   = "VCLUSTER_INGRESS_HOST"
+	DEFAULT_K3S_VERSION         = "rancher/k3s:v1.27.3-k3s1"
+	UCLUSTER_SYNC_PLUGIN_TAG    = "uffizzi/ucluster-sync-plugin:v0.2.4"
+	OCI_TYPE                    = "oci"
+	PREMIUM_RWO_STORAGE_CLASS   = "premium-rwo"
+	NO_SCHEDULE                 = "NoSchedule"
+	SANDBOX_GKE_IO_RUNTIME      = "cloud.google.com/gke-sandbox"
+	EXISTS                      = "Exists"
```

Comment:
probably available as `v1.NodeSelectorOpExists`

Reply:
it is available, I am using it now